kit/layout/src/Portal.tsx

```diff
@@ -35,12 +35,17 @@ function PortalConsumer({ forId, absoluteFill, manager, children }: PortalConsum
         } else {
             manager.update(key.current, children, forId, absoluteFill);
         }
+    }, [manager, children, forId, absoluteFill]);
 
+    React.useState(() => {
         return () => {
+            if (key.current == null) {
+                return;
+            }
             // @ts-ignore
-            manager.unmount(key.current, forId, absoluteFill);
+            manager.unmount(key.current, forId);
         };
-    }, [manager, children, forId, absoluteFill]);
+    });
```

Comment:
I bet you've forgotten an empty array

```suggestion
    }, []);
```